src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -3322,28 +3322,34 @@ internal EffectiveParameters(ImmutableArray<TypeWithAnnotations> types, Immutabl
             return paramRefKind;
         }
 
-        // In method group conversions, 'in' is allowed to match 'ref' and 'ref readonly' is allowed to match 'ref' or 'in'.
-        internal static bool AreRefsCompatibleForMethodConversion(RefKind x, RefKind y, CSharpCompilation compilation)
+        // In method group conversions,
+        // - 'in' is allowed to be converted to 'ref',
```

Comment:
>'in' is allowed to be converted to 'ref', [](http://example.com/codeflow?start=13&length=41)

Same comment as for the speclet update. I don't think the language has a concept conversion for modifiers.